.github/workflows/build_truss_push.yaml

```diff
@@ -1,23 +1,45 @@
 ---
-name: Build Truss Image Push
-'on':
+name: Build and push Truss image
```

Comment:
this will soon be going away as we plan to delete the truss branch, but it has a stay of execution for now.